src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -110,16 +113,18 @@ export default function NftDrawer({
               />
             ))}
           </Space>
-          <Button
-            type="dashed"
-            onClick={() => {
-              setAddTierModalVisible(true)
-            }}
-            style={{ marginTop: 15 }}
-            block
-          >
-            <Trans>Add reward tier</Trans>
-          </Button>
+          {rewardTiers.length < MAX_NFT_REWARD_TIERS ? (
```

Comment:
**suggestion** maybe we `disable` the button instead of removing it. wdyt?

Reply:
Yeah ok